Source/PagesController.swift

```diff
@@ -107,6 +120,11 @@ extension PagesController {
     }
   }
 
+  public func pageViewController(pageViewController: UIPageViewController, didFinishAnimating finished: Bool, previousViewControllers: [AnyObject], transitionCompleted completed: Bool) {
```

Comment:
@kostiakoval I guess in style guidelines we agreed on one line break for functions with long signatures, didn't we?